deployments/docker-compose/docker-compose-interval.yaml

```diff
@@ -0,0 +1,172 @@
+version: "3"
+
+services:
+  postgres:
+    image: postgres
+    container_name: postgres
+    hostname: postgres
+    volumes:
+      - ./postgres/init.sql:/docker-entrypoint-initdb.d/init.sql
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: "trust"
+#  clicker:
+#    image: public.ecr.aws/sumologic/the-coffee-bar:clicker-linux
+#    container_name: clicker
+#    hostname: clicker
+#    environment:
+#      COFFEE_BAR_UI_URL: "http://the-coffee-bar-frontend:3000"
+#      DELAY: "10"
+#      PUPPETEER_PRODUCT: "chrome"
+#      DEBUG_DUMPIO: 'false'
+#    cap_add:
+#      - SYS_ADMIN
+#    command: ["node", "src/clicker.js", "http://the-coffee-bar-frontend:3000"]
+#    depends_on:
+#      - the-coffee-bar
+#      - the-coffee-bar-frontend
+#    ports:
+#      - 9222:9222
+#    expose:
+#      - 9222
+  the-coffee-bar:
+    image: public.ecr.aws/g0d6f4n6/the-coffee-bar-arun:pythonAppsv0.2.1-ot1.7.1-0.26b1
+    container_name: the-coffee-bar
+    hostname: the-coffee-bar
+    ports:
+      - 8082:8082
+    expose:
+      - 8082
+    command: ["opentelemetry-instrument", "the-coffee-bar",
+              "--host=the-coffee-bar", "--port=8082", "--coffeemachine-host=the-coffee-machine",
+              "--coffeemachine-port=8083", "--cashdesk-host=the-cashdesk", "--cashdesk-port=8084"]
+ #             ,"--sweets-url=LAMBDA_URL_HERE"]
+    environment:
+      - OTEL_TRACES_EXPORTER=otlp_proto_http
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otelcol:55681/v1/traces
+      - OTEL_RESOURCE_ATTRIBUTES=application=tcb-local,service.name=the-coffee-bar
+      - OTEL_PROPAGATORS=xray,tracecontext,baggage
+  the-coffee-bar-frontend:
+    image: public.ecr.aws/g0d6f4n6/the-coffee-bar-arun:frontend-jsv0.1.1-ot0.24.0
+    container_name: the-coffee-bar-frontend
+    hostname: the-coffee-bar-frontend
+    ports:
+      - 3000:3000
+    expose:
+      - 3000
+    command: |
+      /bin/bash -c "python3 ./cpu_killer/cpu_killer.py & \
+       npm start"
+    environment:
+      - INTERVAL_BASED_TRIGGER=true
+      - SPIKE_INTERVAL_DAYS=0
+      - SPIKE_INTERVAL_HOURS=1
+      - SPIKE_START_DATE=
+      - SPIKE_CRON=*/2 * * * *
+      - SPIKE_DURATION=60
+      - CPU_SPIKE_PROCESSES=1
+      - CHOKIDAR_USEPOLLING=true
+      - REACT_APP_RUM_SCRIPT_URL=https://rum.sumologic.com/sumologic-rum.js
+      - REACT_APP_COFFEE_BAR_URL=http://localhost:8082/order
+      - REACT_APP_SERVICE_NAME=the-coffee-bar-frontend
+      - REACT_APP_COLLECTION_SOURCE_URL=
+      - REACT_APP_APPLICATION_NAME=tcb-local
+      - REACT_APP_DEFAULT_ATTRIBUTES={"testKey":"testValue","testListKey":["1"]}
+      - REACT_APP_PROPAGATION_CORS_URLS=[/^http:\/\/localhost:8082\/.*/,/^http:\/\/the-coffee-bar:8082\/.*/,]
+  the-coffee-machine:
+    image: public.ecr.aws/g0d6f4n6/the-coffee-bar-arun:pythonAppsv0.2.1-ot1.7.1-0.26b1
```

Comment:
Looks like you used here your private image. Please set the official one

Reply:
Fixed